x/programs/rust/wasmlanche-sdk/src/state.rs

```diff
@@ -68,30 +96,47 @@ impl State {
     /// the host fails to read the key and value.
     /// # Panics
     /// Panics if the value cannot be converted from i32 to usize.
-    pub fn get<T, K>(&self, key: K) -> Result<T, Error>
```

Comment:
🤔... I'm just realizing that this API is flawed. Can you open another PR that fixes the return type here? We don't handle the case where there's nothing stored at the key. This function should return a `Result<Option<T>, Error>`.